queries.sql

```diff
@@ -52,3 +52,19 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, escape_attempts FROM animals WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+-- multi tables query using join
+
+SELECT animals.name FROM animals INNER JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond ';
+
+SELECT animals.name FROM animals INNER JOIN species ON animals.species_id = species.id WHERE species.name = 'Pokemon';
+
+SELECT owners.full_name, animals.name FROM animals RIGHT JOIN owners ON animals.owner_id = owners.id;
```

Comment:
- [ ] To get the list of all owners and their animals, you are supposed to firm select from the owners table and then join the animal's table. KIndly make this change.

![image](https://user-images.githubusercontent.com/86318284/213537613-3e707a82-421a-40df-a964-829c0763db05.png)

Reply:
Thank you for the detailed observation. I have acted on it as required.